.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
should remove all these .idea files (maybe add .idea/ to .gitignore)

Reply:
sure, will do first thing in the morning

Reply:
done!